docs/ops/sort/MulticlassNMS_9.md

```diff
@@ -0,0 +1,175 @@
+##  MulticlassNonMaxSuppression<a name="MulticlassNonMaxSuppression"></a> {#openvino_docs_ops_sort_MulticlassNonMaxSuppression_9}
+
+**Versioned name**: *MulticlassNonMaxSuppression-9*
+
+**Category**: *Sorting and maximization*
+
+**Short description**: *MulticlassNonMaxSuppression* performs multi-class non-maximum suppression of the boxes with predicted scores.
+
+**Detailed description**: *MulticlassNonMaxSuppression* is a multi-phase operation. It implements non-maximum suppression algorithm as described below:
+
+1.  Let `B = [b_0,...,b_n]` be the list of initial detection boxes, `S = [s_0,...,s_N]` be  the list of corresponding scores.
+2.  Let `D = []` be an initial collection of resulting boxes. Let `adaptive_threshold = iou_threshold`.
+3.  If `B` is empty, go to step 9.
+4.  Take the box with highest score. Suppose that it is the box `b` with the score `s`.
+5.  Delete `b` from `B`.
+6.  If the score `s` is greater than or equal to `score_threshold`,  add `b` to `D`, else go to step 9.
+7.  If `nms_eta < 1` and `adaptive_threshold > 0.5`, update `adaptive_threshold *= nms_eta`.
+8.  For each input box `b_i` from `B` and the corresponding score `s_i`, set `s_i = 0` when `iou(b, b_i) > adaptive_threshold`, and go to step 3.
+9.  Return `D`, a collection of the corresponding scores `S`, and the number of elements in `D`.
```

Comment:
nit:
This list is titled as algorithm but there are several places of confusion. 
Where do we stop iterating over boxes.
Why should we delete smth from B?

I don't insist on changing it rn. You could create a ticket for yourself to do it later. And also could we rearrange this algo for previous NMSs also?

Reply:
This algo is the inner part which describes the nms inside a class, as the following sentence addresses: "This algorithm is applied independently to each class of each batch element. "

Actually there is no change in this part from previous. The only change is an extra input format is added.